Workiva/w-mobile-kit
# MOBILE-2413: Fixed issue with sizing issues

Description
---
If a WPanelVC persists but hides and the sizes change, then issues arise when presenting the WPanelVC again

What Was Changed
---
* setupUI is now called on viewWillAppear
* sidePanel property is now a computed property for consistency

Testing
---
* Test on iPad Air 2 or better
   * Open WPanelVC in full screen, open side panel
   * Change away from VC, multitask the screen to half width
   * Go back to panel vc, verify it appears correctly as a vertical panel

---

Please Review: @Workiva/mobile
